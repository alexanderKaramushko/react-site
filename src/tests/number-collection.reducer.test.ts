import { numberCollectionReducer } from '../reducers/number-collection.reducer';

describe('With snapshots ', () => {
    it('+++ reducer with shapshot', () => {
        const action = {
            payload: 1,
            type: '[1] NumberGenerator async service returned a new number.',
        };

        expect(numberCollectionReducer(undefined, action)).toMatchSnapshot();
    });
});
