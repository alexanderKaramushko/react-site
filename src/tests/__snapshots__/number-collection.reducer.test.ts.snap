// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With snapshots  +++ reducer with shapshot 1`] = `
Array [
  0,
  1,
]
`;
